### DISCUSSION TITLE
indexing images

### DISCUSSION THREAD

**Discussion by PrismStrike** [dadoonet/fscrawler#1273] [Q&A] [answered]
I want to index images and exif data of these, so I downloaded the jai-imageio-jpeg2000-1.4.0.jar. Then I moved this file to the "/fscrawler-es7.2.8-SNAPSHOT/lib" folder but it's still not working ...
Do you have a clue for me what I am doing wrong?

![IMG_20211007_084219 (2)](https://user-images.githubusercontent.com/85169331/136334297-25195d43-0fc3-45c6-9fc2-821d7d1d16c2.jpg)
![IMG_20211007_084249 (2)](https://user-images.githubusercontent.com/85169331/136334303-6f8d934f-061e-4584-b163-2b53e72381e2.jpg)

**Comment by dadoonet** [comment, marked as answer]
You are doing nothing wrong. It's a bug sadly. There's a pending PR https://github.com/dadoonet/fscrawler/pull/1271 from @NickUfer which I need to review ASAP and merge.

**Reply by dadoonet** [reply]
I just merged #1271 so I hope this will work well with the next snapshot which is building ATM. Could you confirm?

**Reply by jhult** [reply]
Not the original poster but I installed one of the latest snapshots from [here](https://oss.sonatype.org/content/repositories/snapshots/fr/pilato/elasticsearch/crawler/fscrawler-es7/2.8-SNAPSHOT/) and it seems to have made that error go away in my logs.

**Reply by dadoonet** [reply]
Thanks a lot @jhult !
